packages/app/src/components/CredentialAttributes.tsx

```diff
@@ -3,7 +3,8 @@ import { Heading, LucideIcons, Paragraph, TableContainer, TableRow, XStack, YSta
 import { formatCredentialSubject } from '../utils'
 
 type CredentialAttributesProps = {
-  subject: Record<string, unknown>
+  // biome-ignore lint/suspicious/noExplicitAny: <explanation>
+  subject: Record<string, any>
```

Comment:
Doesn't `unknown` work?

Reply:
no  and it doesn't really matter as it's just JSON. So i decided to go for the easy route